package/MDAnalysis/core/groups.py

```diff
@@ -1818,10 +1824,6 @@ def __init__(self, ix, u):
         self._ix = ix
         self._u = u
 
-    def __repr__(self):
```

Comment:
We are probably going to go back to the terse representation, see #1223 and comments therein.